README.md

```diff
@@ -61,6 +61,8 @@ use: "file-loader?name=[name].[ext]&publicPath=assets/foo/&outputPath=app/images
 }
 ```
 
+`cssOutputPath` also should be used (together with `useRelativePath` property) to define the output path context.
```

Comment:
```
|– src
|    |– css
|         | ...
|– dist 
|     |– bundle.js
|     |– bundle.css
|– webpack.config.js
|– package.json
```

**webpack.config.js**
```
...

{
  loader: 'file-loader'
  options: {
     useRelativePath: true,
     cssOutputPath: '...'
   }
}
...

{
    use: ExtractTextPlugin.extract({
      fallback: 'style-loader'
      use: [ 'css-loader' ]
    })
}
...

new ExtractTextPlugin()
```

**path/to/file.(svg|png|gif|jpg)**
```
// Example Asset
```

**path/to/file.css**
```
// Example CSS
```

**bundle.css**
```
// CSS Bundle (cssOutputPath + ETWP)
```

I came back here later in terms of tests 😛